Wordle.Api/Wordle.Api/Dtos/PlayerDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Wordle.Api.Dtos
+{
+    public class PlayerDto
+    {
+        public string? PlayerName { get; set; }
+        public int GameCount { get; set; }
```

Comment:
You should be able to get away with removing the GameCount field from the DTO because it can be incremented in the service.